tests/ctst/steps/notifications.ts

```diff
@@ -49,41 +76,51 @@ async function emptyVersionedBucket(world: IWorld) {
     }));
 }
 
-async function putObject(world: IWorld) {
+async function putObject(world: Zenko) {
     world.resetCommand();
-    world.addCommandParameter({ bucket: world.saved.bucketName });
-    world.addCommandParameter({ key: world.saved.objectName });
-    world.saved.putResult = await S3.putObject(world.getCommandParameters());
+    world.addCommandParameter({ bucket: world.getSaved().bucketName as string });
+    world.addCommandParameter({ key: world.getSaved().objectName as string });
+    const result = await S3.putObject(world.getCommandParameters());
+    world.setResult(result);
 }
 
-async function copyObject(world: IWorld) {
+async function copyObject(world: Zenko) {
     await putObject(world);
     world.resetCommand();
     let objName = `object-${Utils.randomString()}`.toLocaleLowerCase();
-    if (world.saved.filterType) {
-        objName = world.saved.filterType === 'prefix' ? `${world.saved.objectNamePrefix}${objName}` :
-            `${objName}${world.saved.objectNameSufix}`;
+    if (world.getSaved().filterType) {
+        objName = world.getSaved().filterType === 'prefix' ?
+            `${world.getSaved().objectNamePrefix as string}${objName}` :
+            `${objName}${world.getSaved().objectNameSufix as string}`;
     }
-    world.addCommandParameter({ bucket: world.saved.bucketName });
+    world.addCommandParameter({ bucket: world.getSaved().bucketName as string });
     world.addCommandParameter({ key: objName });
-    world.addCommandParameter({ copySource: `${world.saved.bucketName}/${world.saved.objectName}` });
-    world.saved.objectName = objName;
+    world.addCommandParameter({
+        copySource:
+            `${world.getSaved().bucketName as string}/${world.getSaved().objectName as string}`,
+    });
+    // TODO this rule honestly just seems dumb, especially in this case
+    // eslint-disable-next-line no-param-reassign
+    world.getSaved().objectName = objName;
```

Comment:
We should have a `setSaved()` method in this case, it does not feel standard to edit the returned variable of a `get` function in the same line

The approach before looked more standard